api-extractor/src/TypeScriptHelpers.ts

```diff
@@ -36,9 +56,43 @@ export default class TypeScriptHelpers {
   /**
    * Returns the JSDoc comments associated with the specified node, if any.
    */
-  public static getJsDocComments(node: ts.Node, sourceFile: ts.SourceFile): ts.CommentRange[] {
+  public static getJsDocComments(node: ts.Node,
+    sourceFile: ts.SourceFile,
+    errorLogger: (message: string) => void): string {
+    let jsDoc: string = '';
     // tslint:disable-next-line:no-any
-    return (ts as any).getJsDocComments(node, sourceFile);
+    if ((node as any).jsDoc && (node as any).jsDoc.length > 0) {
+      // tslint:disable-next-line:no-any
+      for (let i: number = 0; i <  (node as any).jsDoc.length; i++) {
```

Comment:
I see you're concatenating multiple JSDoc blocks.  Here's an interesting case to consider for that:

```typescript
/**
 * @file index.ts
 * Copyright Microsoft
 */

/**
 * This is my class
 */
class Blah {
}
```